<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Bienvenida</class>
 <widget class="QDialog" name="Bienvenida">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>633</width>
    <height>268</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::PreventContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Saludar</string>
  </property>
  <widget class="QLabel" name="etq_nombre">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Introduce tu nombre: </string>
   </property>
  </widget>
  <widget class="QLabel" name="etq_saludo">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Saludo:</string>
   </property>
  </widget>
  <widget class="QLabel" name="etq_resultado">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>100</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="texto_nombre">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>40</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="boton_saludar">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>170</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Saludar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
